Neos.ContentRepository.Core/Classes/Feature/WorkspaceCommandHandler.php

```diff
@@ -419,17 +476,18 @@ private function extractCommandsFromContentStreamMetadata(
             if (isset($metadata['commandClass'])) {
                 $commandToRebaseClass = $metadata['commandClass'];
                 $commandToRebasePayload = $metadata['commandPayload'];
-                if (!method_exists($commandToRebaseClass, 'fromArray')) {
+
+                if (array_diff(class_implements($commandToRebaseClass) ?: [], [CommandInterface::class, RebasableToOtherWorkspaceInterface::class]) === []) {
```

Comment:
we now check agains the interface instead of `method_exists($commandToRebaseClass, 'fromArray')`